diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/AvoidEmptyPrimaryConstructor.kt

```diff
@@ -29,8 +26,7 @@ class AvoidEmptyPrimaryConstructor(private val configRules: List<RulesConfig>) :
 
     @Suppress("UnsafeCallOnNullableType")
     private fun checkCLass(node: KtClass) {
-        println(node.node.prettyPrint())
-        if(node.primaryConstructor?.valueParameters?.isNotEmpty() != false || node.primaryConstructor?.isPrivate() == true)
+        if(node.primaryConstructor?.valueParameters?.isNotEmpty() != false || node.primaryConstructorModifierList != null)
```

Comment:
Can `primaryConstructorModifierList` be non-null, but empty? Is `?.isEmpty() == true` check needed?